docs/releases/5.0.txt

```diff
@@ -150,14 +150,14 @@ form::
     ]
 
 
-    class Winners(models.Model):
+    class Winner(models.Model):
         name = models.CharField(...)
         medal = models.CharField(..., choices=Medal.choices)
         sport = models.CharField(..., choices=SPORT_CHOICES)
 
-Django 5.0 supports providing a mapping instead of an iterable, and also no
-longer requires ``.choices`` to be used directly to expand :ref:`enumeration
-types <field-choices-enum-types>`::
+Django 5.0 added support for accepting a mapping or a callable instead of an
```

Comment:
Use present tense when referring to the version being released in its release notes:

```suggestion
Django 5.0 adds support for accepting a mapping or a callable instead of an
```